explorer/explorerroutes.go

```diff
@@ -93,6 +93,16 @@ func (exp *explorerUI) Block(w http.ResponseWriter, r *http.Request) {
 		exp.ErrorPage(w, "Something went wrong...", "could not find that block", true)
 		return
 	}
+	var count int
+	data.TxAvailable = true
+	for _, i := range data.Tx {
+		if i.Coinbase == true {
```

Comment:
Coinbase is already a boolean.

explorer/explorerroutes.go:99:6:warning: should omit comparison to bool constant, can be simplified to i.Coinbase (S1002) (gosimple)